### PR TITLE
CSS build support and Avatar CSS conversion

### PR DESCRIPTION
**Pull request by jaketrent**
Fixed JS->CSS build which had been broken since ESM. Added scripts to Build for reuse. Exposed refactor helper in Core. Used Storybook postcss preset. 

What do you think of the utils?
How about the conversion itself?

### CODE REVIEW

File: packages/avatar/src/css/index.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  object-fit: cover;
+}
+.psds-avatar__initials {
+  width: auto;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  color: var(--psColorsTextIconHighOnDark);
+  /* TODO: revert to bg util color after it exists in css vars */
```
Comment (jaketrent) [inline packages/avatar/src/css/index.css:53]: see https://github.com/pluralsight/design-system/issues/1817

**Comment by EdwardIrby** [comment]
@jaketrent we should probably have @sweeting look at these going forward for visual errors that test can not catch.

**Review by EdwardIrby — CHANGES_REQUESTED** [review]
Seeing the following warning message when running build in Avatar
`You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.`

Further we need to build to a path relative to the index.js files in the esm and cjs directories. 
- `dist/cjs/css/index.css`
- `dist/esm/css/index.css`

Storybook wouldn't show these failing nor our test but for the distributed packages we want them at these locations.

**Comment by EdwardIrby** [comment]
Visually looks the same as the glamor variant!

**Comment by jaketrent** [comment]
@EdwardIrby Thanks for the review. Good catch on the postcss plugin missing. 

I'm not thrilled with having the css under the `dist/esm` and `dist/cjs` subdirs. The `index.css` file is not of that tech and thus seems wrongly categorized. I know that this is for the `dist` dir to be able to relative import like the `src` dir. I don't have another solution in mind yet.

**Pull request merged**